tests/baselines/reference/importAssertionNonstring.errors.txt

```diff
@@ -16,14 +32,26 @@ mod.mts(11,66): error TS2837: Import assertion values must be string literal exp
 !!! error TS2837: Import assertion values must be string literal expressions.
     
     import * as thing3 from "./mod.mjs" assert {field: /a/g};
+                                        ~~~~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type '{ field: RegExp; }' is not assignable to type 'ImportAttributes'.
+!!! error TS2322:   Property 'field' is incompatible with index signature.
+!!! error TS2322:     Type 'RegExp' is not assignable to type 'string'.
                                                        ~~~~
 !!! error TS2837: Import assertion values must be string literal expressions.
     
     import * as thing4 from "./mod.mjs" assert {field: ["a"]};
+                                        ~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type '{ field: string[]; }' is not assignable to type 'ImportAttributes'.
+!!! error TS2322:   Property 'field' is incompatible with index signature.
+!!! error TS2322:     Type 'string[]' is not assignable to type 'string'.
                                                        ~~~~~
 !!! error TS2837: Import assertion values must be string literal expressions.
     
     import * as thing5 from "./mod.mjs" assert {field: { a: 0 }};
+                                        ~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type '{ field: { a: number; }; }' is not assignable to type 'ImportAttributes'.
+!!! error TS2322:   Property 'field' is incompatible with index signature.
+!!! error TS2322:     Type '{ a: number; }' is not assignable to type 'string'.

```

Comment:
It’s worth considering the pros and cons of applying the checking to import assertions as well as import attributes. My instinct was to suggest leaving them alone with no checking, but I don’t know that I want to give people a reason _not_ to migrate to import attributes.

Reply:
@andrewbranch Thanks for the feedback. Is it okay to skip checking _import assertions_, or should I change `getGlobalImportAttributesType` to handle both the deprecated `ImportAssertions` and `ImportAttributes`?

Reply:
I don’t think the type of import attributes should be affected by the `ImportAssertions` type—I was just wondering whether new errors should appear at all on import _assertions_. I’m not sure, so I raised this comment for other team members to consider while reviewing. I’m not suggesting any change at this point 👍

Reply:
oke, got it.

Reply:
Conclusion from design meeting was this is fine the way it is.

Reply:
@andrewbranch Oke, thanks for the feedback.